lib/sassc/rails/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module SassC
   module Rails
-    VERSION = "3.0.0"
+    VERSION = '4.0.0'
```

Comment:
```suggestion
    VERSION = '3.1.0'
```

This should be bumped as a minor version instead of major version.

Although this is against semver practice, in reality this is how ruby gems bump gem version when dropping old ruby, even gems maintained by the ruby team does the same. The reason is that rubygems is smart enough to pick the highest version of gem that met the `required_ruby_version`, therefore user with older ruby version will automatically get old version of gem and users with newer ruby will get new version. From a user experience perspective, nothing will be broken like npm (because npm does not enforce engine version requirement like rubygems), therefore bumping it as minor version would help users with new ruby receive updates without much effort if they have already locked the minor version e.g. `~> 3.0`.

Reply:
OK, we will go with a minor version here. 👍 Not strictly semver but since nothing is expected to break, its ok.